src/transformState.js

```diff
@@ -5,7 +5,38 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
add here all properties instead of using destructuring assignment in the each case